src/cdomain/value/cdomains/intDomain.mli

```diff
@@ -77,13 +77,13 @@ sig
 
   (** {b Logical operators} *)
 
-  val lognot: t -> t
+  val c_lognot: t -> t
   (** Logical not: [!x] *)
 
-  val logand: t -> t -> t
+  val c_logand: t -> t -> t
   (** Logical and: [x && y] *)
 
-  val logor : t -> t -> t
+  val c_logor : t -> t -> t
   (** Logical or: [x || y] *)
 
 end
```

Comment:
This is one of the places where comments and function names should be made to match.

Reply:
This matches though. The `c_` prefix means it's logical in the C sense, which is what the comment also says.

Reply:
Confusing how? They still correctly refer to what kind of C operations they're for and include example C expressions.

The confusing thing is OCaml using `log` prefix for bitwise operations and calls them "bitwise logical" in standard library documentation. All we can do is be consistent with it.